### PR TITLE
fix(progress-indicator): fix active state

### LINKED ISSUE #9733: [Bug]: ProgressIndicator – interactive progress indicator steps do not receive :active state
### Package

carbon-components, carbon-components-react

### Browser

Chrome

### Package version

10.44.1/7.44.1

### Description

Ref https://github.com/carbon-design-system/carbon/pull/9631#issuecomment-919996095

Interactive / clickable steps that do not have a tooltip don't receive a visual `:active` state anymore.

Screen Recording 2021-09-24 at 11 45 51

### CodeSandbox example

https://react.carbondesignsystem.com/?path=/story/components-progressindicator--interactive

### Steps to reproduce

Drag any interactive / clickable step that does not have a tooltip in the react storybook.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #9733 

This PR adds the :active state to interactive steps that do not have a tooltip. Checkout original issue for full details. 

- [ ] Make sure interactive step that _does_ not have tooltip has `:active`

### CODE REVIEW

**Review by aledavila — APPROVED** [review]

**Comment by janhassel** [comment]
Testing out the storybook preview this seems to also add the `:active` state on non-interactive steps ("Really long label" in the story) again. Ref #9609

Would `.bx--progress-step-button:not(.bx--progress-step-button--unclickable) .bx--progress-label` be an appropriate selector for this? If `:not()` is discouraged, I think there would need to be a class `.bx--progress-step-button--clickable` to identify interactive steps of any kind.

**Review by joshblack — APPROVED** [review]

**Review by joshblack — DISMISSED** [review]
Ah, wrong button click 🤦  Quick +1 to Jan's comments above

File: packages/components/src/components/progress-indicator/_progress-indicator.scss [hunk 1]
```diff
@@ -102,7 +102,8 @@
     margin-right: 0.75rem;
   }
 
-  .#{$prefix}--tooltip__label.#{$prefix}--progress-label:active {
+  .#{$prefix}--progress-step-button:not(.#{$prefix}--progress-step-button--unclickable)
+    .#{$prefix}--progress-label:active {
```
Comment (jnm2377) [inline packages/components/src/components/progress-indicator/_progress-indicator.scss:106]: looks good, but can you add this fix to the `@carbon/styles` package as well?

**Review by jnm2377 — CHANGES_REQUESTED** [review]

**Review by theiliad — APPROVED** [review]

**Review by jnm2377 — APPROVED** [review]

File: packages/styles/scss/components/progress-indicator/_progress-indicator.scss [hunk 2]
```diff
@@ -225,6 +225,12 @@ $progress-indicator-bar-width: 1px inset transparent !default;
     cursor: pointer;
   }
 
+  .#{$prefix}--progress-step-button:not(.#{$prefix}--progress-step-button--unclickable)
+    .#{$prefix}--progress-label:active {
+    box-shadow: 0 rem(3px) 0 0 $interactive-01;
+    color: $interactive-01;
```
Comment (jnm2377) [inline packages/styles/scss/components/progress-indicator/_progress-indicator.scss]: it doesn't seem to like this token. Is this a v11 token?

**Review by jnm2377 — CHANGES_REQUESTED** [review]

File: packages/styles/scss/components/progress-indicator/_progress-indicator.scss (hunk 2 above)
Comment (andreancardona) [reply packages/styles/scss/components/progress-indicator/_progress-indicator.scss]: @jnm2377 Just saw this also...looking into it now

**Review by jnm2377 — APPROVED** [review]

**Pull request merged**